### DISCUSSION TITLE
Staying within a newly opened window

### DISCUSSION THREAD

**Discussion by lmburns** [sayanarijit/xplr#234] [General]
've been trying for a minute to try and use a simple command line `dust` or to use `bat` as a pager to view logs and I can't get the new window to stay open. Do you know which `message` I would use to do this? 
```lua
key.S = {
  help = "folder sizes",
  messages = {
    { BashExec = [[dust]] },
    "PopMode",
  },
}
```
Also, what would `[===[dust]===]` do instead so I know to use it properly?
The `bat` example is the following. If less is used it works, but using `bat --paging=always` does not seem to stay in the window.
```lua
actkey.l = {
  help = "logs",
  messages = {
    { BashExec = [[ cat -- "${XPLR_PIPE_LOGS_OUT}" | less -+F ]] },
    "PopMode",
  },
}
```
Thanks

**Comment by sayanarijit** [comment]
As for `dust`, try `dust | less`, or `dust && read -p "[enter to continue]"`.

`[[foo]]`, `[=[foo]=]`, `[===[foo]===]` and `"foo"` all will work. The `[===[` syntax works better for escaping quotes. `[[` is reserved in bash as condition operators. So, using conditions inside `[[` block will get difficult. I go for the `[===[` syntax because it provides the highest level of convenience for escaping and using bash operators. And, also it keeps the format consistent across the whole script.

As for `bat`, I'm not sure but `PAGER="bat --paging=always" xplr` works fine for me. Please check if there's any customization.
